apps/sms/test/marionette/lib/views/dialer.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+
+var View = require('./view');
+var Dialer =  require('../../../../../../apps/communications/dialer/test/marionette/lib/dialer.js');
+
+function DialerWrapper(client) {
+  this.client = client;
+}
+module.exports = DialerWrapper;
+
+DialerWrapper.prototype = {
+  __proto__: View.prototype,
```

Comment:
I don't have a strong opinion on this. Why do you think it looks better? To me, defining the prototype prevented the repetition of `DialerWrapper.prototype` each time I needed to define a method.